source/fma.f90

```diff
@@ -515,7 +519,7 @@ end function tunenaff
 
               ! Write normalised particle amplitudes
               ! (only when reading physical coordinates)
-              if (fma_writeNormDUMP .and. .not.hasNormDumped(j) ) then
+              if(fma_writeNormDUMP .and. .not.hasNormDumped(j) ) then
```

Comment:
This was definitely the cause of the problem as the conditions were different for opening and writing the file. The error was that it tried to open a negative file unit. I.e. none was assigned and we had some random memory value as unit.

This can still probably be improved a bit. The open call should definitely be with the request call.

Regarding whether NORM dumps should be removed, I think should be addressed in a separate PR.